src/protocol.h

```diff
@@ -400,9 +400,11 @@ enum GetDataMsg : uint32_t {
 /** inv message data */
 class CInv
 {
+private:
```

Comment:
Personally I don't really know if it makes sense for "plain-data" network message types like CInv to have private fields.

Reply:
Oh ok, in that case we may want to make CInv a struct to avoid confusion. As a class, it looked like encapsulation was desirable, and only one place in the code was still accessing `CInv::type` from the outside so it was easy to make it private.

Reply:
Er, renaming `CInv` to `Inv` to make it a struct would be too much noise. Done, dropped the last commit "p2p: refactor CInv::type from public to private class member".